administrator/components/com_media/resources/scripts/app/Api.js

```diff
@@ -106,16 +106,19 @@ class Api {
      * @TODO DN improve error handling
      */
     _handleError(error) {
-        alert(error.status + ' ' + error.statusText);
+        if (error.status !== 200) {
+            alert(error.status + ' ' + error.statusText);
+        }
+        var url_array = window.location.href.split('/');
         switch (error.status) {
             case 404:
                 break;
             case 401:
             case 403:
             case 500:
-                window.location.href = '/administrator';
+                window.location.href = '/' + url_array[3] + '/administrator';
```

Comment:
This will only work for joomla instances which are installed in a subfolder (my-domain.tld/subfolder/), but will not work for joomla instances on the root level (my-domain.tld) or sub-subfolders (my-
domain.tld/subfolder/another-subfolder/).